### PR TITLE
[Bugfix:InstructorUI] Fixed PDF Annotation Toolbar

### LINKED ISSUE #7323: PDF Annotation Toolbar Breaks on Horizontal Scroll
**Describe the bug**
While being zoomed in on a PDF in the grading interface, when trying to scroll to the right, the toolbar doesn't stay at the top as expected which makes some of the buttons no longer accessible (see screenshots)

before scrolling:
pdf_anontate_before

after scrolling:
pdf_anotate_after

### PR DESCRIPTION
**Pull request by immortalcodes**
### What is the current behavior?
 Closes #7323:While being zoomed in on a PDF in the grading interface, when trying to scroll to the right,the tools available gets hidden and inaccessible.  

### What is the new behavior?
Zoomed in on a PDF in the grading interface, when trying to scroll to the right, the position of the toolbar doesn't change and hence the tools don't hide and are always available.
### Other information?
- [x] Tested locally

### CODE REVIEW

**Comment by akshatbatra** [comment]
Annotation toolbar now overlays the document, hiding some of its contents along with hiding the top portion of vertical scrollbar.

**Comment by immortalcodes** [comment]
It doesn't hide the document unless you zoom it and scroll down and then to show some part , some part will be hidden. Am I getting something wrong ? May be u could attach some screenshots for the same.

**Comment by akshatbatra** [comment]
Submitty-panel

**Comment by immortalcodes** [comment]
@akshatbatra I have done some changes , could you verify if the issue is happening again?

**Review by akshatbatra — COMMENTED** [review]
Does the added function run only once (on page load) ? If that's the case then resizing the window can lead to unexpected behavior, you can fix it by adding a resize event listener. But if that's not the case then the changes seem good.

**Review by akshatbatra — COMMENTED** [review]
`mouseover` is a working hack here. I think there is an extra `$( ... )` enclosing the event listener.  I think most browsers don't support `onresize` attribute for all HTML elements, it's only supported for `window` object. You can get rid of the `onresize`. if you want to apply a fix that aligns more with the problem, then you can go with `ResizeObserver`. But IE isn't happy with that.

File: site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig [hunk 1]
```diff
@@ -53,4 +53,11 @@
             $('#pdf-buttons').append(document.getElementById('download-annotations-btn-template').content.cloneNode(true));
         }
     });
+
+    $($('#submission_browser').on("mouseover",function addOffset(){
```
Comment (akshatbatra) [inline site/app/templates/grading/electronic/PDFAnnotationEmbedded.twig:57]:
```suggestion
    const offSetter = new ResizeObserver(entries => {
        entries.forEach(entry => {
            let height = $(".sticky-file-info").outerHeight();
            $("#viewer").css("paddingTop", height);
        });
    });
    offSetter.observe(document.querySelector(".sticky-file-info"));
```

**Review by williamjallen — COMMENTED** [review]
I'm a little confused as to why this PR is using JS at all.  Unless I'm misunderstanding the issue, I believe that it can be resolved entirely with CSS styling.

**Comment by immortalcodes** [comment]
I tried doing it with CSS right from the start, but there were a lot of problems. I implemented a lot of solutions but each one had it's own issue when tested with resizing of the window, different resolutions or zooming(in/out) the pdf . Sometimes the toolbar overlapped the thing or sometimes the resizing created problems.
So I used JS to add a dynamic offset for the annotation pdf which adjusts itself on resizing or change of screen resolution.
I think I have tried most possible options but then still if there are some more suggestions I can give it a shot.

**Review by Eceptonsu — COMMENTED** [review]
Works for me.

**Pull request merged**